runtime/src/context.rs

```diff
@@ -57,6 +57,7 @@ impl Context {
 
     /// Set the context error
     pub fn set_error(&mut self, e: impl std::fmt::Debug) {
+        trace!("Set context error: {:?}", e);
```

Comment:
would it make sense to use `error!` here? or do we likely want to catch that at the source of the error instead (and  using `trace!` here is best for the intentionally verbose debugging level?)

Reply:
I believe the error is being logged somewhere else, this log was added while debugging some issues with the error message itself.